core/tests/org.openjdk.jmc.flightrecorder.test/src/test/java/org/openjdk/jmc/flightrecorder/test/internal/util/ParserToolkitTest.java

```diff
@@ -0,0 +1,26 @@
+package org.openjdk.jmc.flightrecorder.test.internal.util;
```

Comment:
This file needs a copyright header and it needs to be moved to src/main from src/test as the project structure has been modified a little lately with https://github.com/openjdk/jmc/commit/6aeee1375d8ea5739ba2a85675fb5e00f1eea9af

Reply:
Ok. I moved it to the indicated location. However, this breaks the IDE integration for IntelliJ but you are probably aware of that.

Reply:
It's not because of style - apparently, Intellij has some assumptions about the source structure based on the gradle structure (eg. you should have tests in a test source root etc.) and having the tests in the main source root makes compilation and running such tests in IDE to fail miserably :(